library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -100,6 +101,9 @@
 
   private boolean monthsReverseOrder;
 
+  private StringBuilder monthBuilder = new StringBuilder(50);
+  private Formatter monthFormatter;
```

Comment:
these could be final. though, the other existing fields aren't, so it's probably not something that bothers anybody.

Reply:
I will set `monthBuilder ` as final, but `monthFormatter` can't be final because we need to set after creating `CalendarPickerView`.